sample-xml/app/src/main/java/co/nimblehq/sample/xml/ui/screens/home/HomeViewModel.kt

```diff
@@ -12,39 +13,33 @@ import javax.inject.Inject
 
 @HiltViewModel
 class HomeViewModel @Inject constructor(
-    private val getModelsUseCase: GetModelsUseCase,
+    dispatchers: DispatchersProvider,
+    getModelsUseCase: GetModelsUseCase,
     private val isFirstTimeLaunchPreferencesUseCase: IsFirstTimeLaunchPreferencesUseCase,
```

Comment:
Since we've updated `getModelsUseCase` to not be a `val` anymore, we might as well update these too (and other `HomeViewModel` classes) for consistency, what do you think? 😄 

The rest LGTM 💪🏻

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/441/commits/4ca6e75bde07ff851340dae3aca03fa74e9ab2b8